debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/StreamingAdapter.java

```diff
@@ -31,4 +31,6 @@ StreamingChangeEventSource getSource(OracleConnectorConfig connectorConfig, Offs
                                          OracleConnection connection, EventDispatcher<TableId> dispatcher, ErrorHandler errorHandler, Clock clock,
                                          OracleDatabaseSchema schema, OracleTaskContext taskContext, Configuration jdbcConfig,
                                          OracleStreamingChangeEventSourceMetrics streamingMetrics);
+
+    boolean getTablenameCaseInsensitivity(OracleDatabaseVersion databaseVersion);
```

Comment:
Does the db version need to be passed as a parameter? Cannot the version be passed as a part of initialization?

Reply:
Unfortunately the adapter is instantiated inside the connector configurations ctor and the database version requires having opened at least one connection to the database, so I'm not sure that's feasible but I'm open to suggestions.

Reply:
@Naros Just add a `configure` or `init` method as is in other cases

Reply:
Done.